decidim-elections/lib/decidim/votings/admin_engine.rb

```diff
@@ -71,63 +72,73 @@ class AdminEngine < ::Rails::Engine
+          menu.add_item :edit_voting,
+                        I18n.t("info", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        decidim_admin_votings.edit_voting_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_votings.edit_voting_path(current_participatory_space)),
+                        if: allowed_to?(:update, :voting, voting: current_participatory_space)
+          menu.add_item :edit_voting_landing_page,
+                        I18n.t("landing_page", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        decidim_admin_votings.edit_voting_landing_page_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_votings.voting_landing_page_path(current_participatory_space)),
+                        if: allowed_to?(:update, :voting, voting: current_participatory_space)
+
+          menu.add_item :components,
+                        I18n.t("components", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        decidim_admin_votings.components_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_votings.components_path(current_participatory_space)),
+                        if: allowed_to?(:read, :component, voting: current_participatory_space),
+                        submenu: { target_menu: :admin_votings_components_menu, options: { container_options: { id: "components-list" } } }
+
+          menu.add_item :attachments,
+                        I18n.t("attachments", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        "#",
+                        if: allowed_to?(:read, :attachment_collection) || allowed_to?(:read, :attachment),
+                        submenu: { target_menu: :decidim_votings_attachments_menu }
+
+          menu.add_item :voting_polling_stations,
+                        I18n.t("polling_stations", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        decidim_admin_votings.voting_polling_stations_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_votings.voting_polling_stations_path(current_participatory_space)),
+                        if: !current_participatory_space.online_voting? && allowed_to?(:read, :polling_stations)
+
+          menu.add_item :voting_polling_officers,
+                        I18n.t("polling_officers", scope: "decidim.votings.admin.menu.votings_submenu"),
+                        decidim_admin_votings.voting_polling_officers_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_votings.voting_polling_officers_path(current_participatory_space)),
+                        if: !current_participatory_space.online_voting? && allowed_to?(:read, :polling_officers)
+          menu.add_item :voting_monitoring_committee_members,
```

Comment:
```suggestion

          menu.add_item :voting_monitoring_committee_members,
```

Reply:
my understanding is that you requested to add a new line.